pkgs/tools/misc/mongodb-tools/assertions.nix

```diff
@@ -0,0 +1,19 @@
+{ goPackages, fetchurl, callPackage }:
```

Comment:
I had some problems with circular dependencies (assertions required by goconvey, and also using goconvey), which made me try different versions. This one is actually not required. Resubmit?